man/fi_domain.3.md

```diff
@@ -375,6 +378,14 @@ are defined.
   manual progress may still need application assistance to process
   received operations.
 
+*FI_PROGRESS_CONTROL_UNIFIED*
+: This progress model indicates that the user will synchronize progressing the
+  data and control operations themselves (i.e.. this allows the control interface
```

Comment:
I'm going to be annoying and comment about this extra period. Sorry.

Reply:
done